install_config/install/advanced_install.adoc

```diff
@@ -508,11 +508,10 @@ which may cause invalid configurations. Example usage:
 a|This variable configures additional `docker` options within
 *_/etc/sysconfig/docker_*, such as options used in
 xref:../../install_config/install/host_preparation.adoc#managing-docker-container-logs[Managing Container Logs].
-Use `json-file` or `journald`. The default is `journald`.
+Use `json-file`.
```

Comment:
How about: "The following example shows the configuration of Docker to use the `json-file` log driver, where Docker will rotate between three (3) 1 MB log files."